openbb_terminal/stocks/stocks_helper.py

```diff
@@ -17,13 +17,15 @@
 import plotly.graph_objects as go
 import pyEX
 import pytz
+import requests
+
 import yfinance as yf
 from alpha_vantage.timeseries import TimeSeries
 from numpy.core.fromnumeric import transpose
 from plotly.subplots import make_subplots
 from scipy import stats
 
-from openbb_terminal.config_terminal import theme
+from openbb_terminal.config_terminal import theme, API_POLYGON_KEY
```

Comment:
let's load config_terminal as cfg and then use `cfg.API_POLYGON_KEY`. That's how we doing in the other places

Reply:
I honestly don't like implicit imports. I think having `cfg.theme` is much more visible where that comes from, and would rather us doing that

Reply:
just tryna use your code against you ;)

Reply:
that also imports everything and is why we initially had such slow load times

Reply:
that you are <3

now approve